examples/fabric/v4/cloudRouterConnectivity/routing-protocol-bgp/README.md

```diff
@@ -12,18 +12,21 @@ At minimum, you must set below variables in `terraform.tfvars` file:
   obtained same way as above
 - `rp_name`- Name of routing Protocol
 - `rp_type`- Type of routing Protocol entity, "BGP"
+- `bgp_enabled_` - Enable or Disable BGP ipv4 and ipv6 sessions
 - connection_uuid = "d557cb4c-9052-4298-b5ca-8a9ed914cf03"
   rp_type = "DIRECT"
```

Comment:
This and the following lines all need leading - marks for list styling and surrounding ` marks for property name styling.